docs/_plugins/copy_javadoc_to_docs.rb

```diff
@@ -10,7 +10,11 @@
 puts "Making directory " + dest
 mkdir_p dest
 
-puts "cp -r " + source + "/. " + dest
-cp_r(source + "/.", dest)
+if !File.directory?(source)
+  puts source + " not found, continuing without javadoc"
```

Comment:
should this be prefixed with something like "WARNING: " ?

Reply:
good idea, added